src/app/plugins/scores.plugin.ts

```diff
@@ -62,81 +72,155 @@ export class ScoresPlugin extends Plugin {
       let teamFound = false;
 
       for (const game of games) {
+        //Stop if the max amount of messages have been queued
+        if (embedBuffer.length >= this._MAX_NUM_DISPLAY) {
+          break;
+        }
+        if (game.includes('DELAYED')) {
+          continue;
+        }
 
-        if (!this._containsAllTokens(game, teamArg.split(' '))) { continue; } //Check if game does not contain all search terms
-        if (embedBuffer.length >= this._MAX_NUM_DISPLAY) { break; } //Stop if the max amount of messages have been queued
-
-        teamFound = true; //Designates as true if any game was found with matching search term
-
-        let visitorData = game.match(visitorDataRegex);
-        let homeData = game.match(homeDataRegex);
+        let visitorData = game.match(visitorDataActiveRegex);
+        let homeData = game.match(homeDataActiveRegex);
 
         //Makes sure regex found data
-        if (visitorData == null || homeData == null) { continue; }
-
-        visitorData = visitorData[0];
-        homeData = homeData[0];
+        if (!visitorData || !homeData) {
+          //If this didn't work, it may be an upcoming game
+
+          //Call function and make sure teamFound is true if this is true atleast once
+          if (this._tryUpcomingGame(embedBuffer, game, teamArg)) {
+            teamFound = true;
+          }
+          continue;
+        }
 
-        visitorData = visitorData.split('%20');
-        homeData = homeData.split('%20');
+        visitorData = visitorData[0].split('%20');
+        homeData = homeData[0].split('%20');
 
         visitorData[0] = visitorData[0].slice(1); //Remove the prefix that starts the visitors name
 
         const visitorName = this._getTeamNameFromTeamData(visitorData);
         const homeName = this._getTeamNameFromTeamData(homeData);
 
+        //Check if game does not contain all search terms
+        if (
+          !(
+            this._strictlyContainsAllTokens(visitorName, teamArg) ||
```

Comment:
should be && 

'if they don't _both_ contain all tokens' => if ( !(first && second ) )

Reply:
It checks to see if one of the teams, contains the search term, not both. If either are true i do not want to skip

Reply:
gotcha, nvm

Reply:
I changed the comment to represent that a little better